manticore/core/workspace.py

```diff
@@ -28,12 +28,12 @@ def __exit__(self, *excinfo):
 import errno
 import threading
 from ..utils import config
-from ..utils.helpers import PickleSerializer
+from ..utils.helpers import PickleSerializer, StateSerializer
```

Comment:
StateSerializer doesn't seem to be used elsewhere in this file

Reply:
Good catch!  That must have been forgotten from an earlier edit.  We don't have a linter on unused imports, so here it is.

```suggestion
from ..utils.helpers import PickleSerializer
```